bluebottle/utils/admin.py

```diff
@@ -91,10 +94,34 @@ def export_as_csv(modeladmin, request, queryset):
         writer = csv.writer(response)
 
         if header:
-            writer.writerow(labels if labels else field_names)
+            row = labels if labels else field_names
+            # For project check if we have extra fields
+            if queryset.model is Project:
+                for field in CustomProjectFieldSettings.objects.all():
+                    labels.append(field.name)
+            if queryset.model is Member:
+                for field in CustomMemberFieldSettings.objects.all():
+                    labels.append(field.name)
+            writer.writerow(row)
 
         for obj in queryset:
-            writer.writerow([prep_field(request, obj, field, manyToManySep) for field in field_names])
+            row = [prep_field(request, obj, field, manyToManySep) for field in field_names]
+            # Write extra field data
+            if queryset.model is Project:
+                for field in CustomProjectFieldSettings.objects.all():
```

Comment:
Tying a general export function to project specific models seems like a bad idea. Any chance we can clean this up a bit?

Reply:
True